src/network/clientpackethandler.cpp

```diff
@@ -1183,6 +1183,29 @@ void Client::handleCommand_HudSetSky(NetworkPacket* pkt)
 	m_client_event_queue.push(event);
 }
 
+void Client::handleCommand_HudSetClouds(NetworkPacket* pkt)
```

Comment:
`HudSetClouds` mirrors the (already existing) `HudSetSky`. So `CloudParams` is probably more fitting than `SkyParams`?